### ISSUE TITLE
qgis_session_info: GRASS7 not found

### ISSUE THREAD

**Issue by pat-s** [jannes-m/RQGIS#85]
```
Error in py_run_string_impl(code, local, convert) : 
  OSError: [Errno 2] No such file or directory: '/Applications/GRASS-7'
```

On my fresh `high_sierra` VM. `run_qgis` works fine. `grass7` is installed. I assume a link problem for now. Investigating later. 

We should write a test for `qgis_session_info()`.

**Comment by jannes-m** [comment]
There is a `qgis_session_info()` test:
 https://github.com/jannes-m/RQGIS/blob/master/tests/testthat/test-qgis-prerun.R

**Comment by pat-s** [comment]
Oh, was not aware of it :smile: 

Then I have to find out whats missing in my VM setup as everything works on Travis :+1:

**Comment by pat-s** [comment]
It works if `grass7` is unlinked, i.e. `brew unlink grass7`. 
If `grass7` is linked, it seems that it is being searched in `/Applications/Grass-7` and not found. 

However, the output of `qgis_session_info()` still does not detect `grass7`  [macOS 10.13.1] :

```
$qgis_version
[1] "2.18.14"

$gdal
[1] "2.2.2"

$grass6
[1] FALSE

$grass7
[1] FALSE

$saga
[1] "2.3.2"

$supported_saga_versions
[1] ""
```

Furthermore, we should update our code so that SAGA v2.3.2 is within the supported SAGA versions.

**Comment by jannes-m** [comment]
Can't tell you much about the MacOS stuff... Internally, QGIS tries to run a GRASS algorithm, if this doen't work, it returns `FALSE`. Can you run GRASS with the help of **RQGIS**, I would suppose not, so that's why there is a `FALSE`.

Regarding the supported SAGA versions, why should we hardcode something? We don't even know if previous SAGA versions are supported or not. That's why the function returns `""`. With the release of QGIS 3, we will simply delete `supported_saga_versions` from the `qgis_session_info()` output.

**Comment by pat-s** [comment]
> With the release of QGIS 3, we will simply delete supported_saga_versions from the qgis_session_info() output.

Okay. I think we can also delete `grass6` as it is not available anymore in QGIS 2.99

### Summary

`grass7` needs to be unlinked otherwise GRASS algs can not be run on macOS. Everything seems to work fine on macOS 10.13.1
